### PR TITLE
includes/footer: make contribute button link to blob not edit.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
This reduces the amount of spam we see, based on other sites.

### CODE REVIEW

**Review by kenyonj — APPROVED** [review]
👍🏼

**Review by BISHOy2580 — COMMENTED** [review]
_includes/footer.html

**Comment by BISHOy2580** [comment]
_includes/footer.html

**Review by Mikec90 — COMMENTED** [review]
Updated 👍🏻
<a data-proofer-ignore href="https://github.com/{{ site.github.repository_nwo }}/edit/main/{{ page.path }}"

File: _includes/footer.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
           <h3 class="h3-mktg mb-3">{{ t.footer.contribute.heading }}</h3>
           <p class="mb-3 p-large">{{ t.footer.contribute.description }}</p>
           <p>
-            <a data-proofer-ignore href="https://github.com/{{ site.github.repository_nwo }}/edit/main/{{ page.path }}"
+            <a data-proofer-ignore href="https://github.com/{{ site.github.repository_nwo }}/blob/main/{{ page.path }}"
```
Comment (Surachai1994) [inline _includes/footer.html:13]: blob

Comment (Surachai1994) [inline _includes/footer.html:13]: blob

**Pull request merged**